addons/website_sale/static/src/interactions/checkout.js

```diff
@@ -72,17 +72,16 @@ export class Checkout extends Interaction {
         // Highlight the newly selected address card.
         this._highlightAddressCard(newAddress);
         const selectedPartnerId = newAddress.dataset.partnerId;
-        await this.waitFor(this.updateAddress(addressType, selectedPartnerId));
+        let addressTypes = [addressType];
         // A delivery address is changed.
         if (addressType === 'delivery' || this.billingContainer.dataset.deliveryAddressDisabled) {
```

Comment:
this is a deadcode as well I suppose?
```suggestion
        if (addressType === 'delivery') {
```